templates/server/index.html

```diff
@@ -1,460 +1,484 @@
 {% extends "templates/base.html" %}
 
 {% block title %}服务器 {{ page_title }}{% endblock %}
-{% block meta_description %}{{ meta_description }}{% endblock %}
+{% block meta_description %}Ubuntu 服务器下载 Ubuntu 20.04 server Ubuntu服务器安装 服务器部署{{ meta_description }}{% endblock %}
 
 {% block content %}
 
-<div class="p-strip is-deep is-bordered">
-    <div class="row">
-      <div class="col-12">
-        <h1>使用 Ubuntu Server 横向扩展</h1>
-      </div>
-    </div>
-    <div class="row u-equal-height">
-      <div class="col-6">
-        <p>Ubuntu Server 可让您的公共或私有数据中心在经济和技术上都具有出色的可扩展性。无论是部署 OpenStack 云、Hadoop 集群还是 50,000 个节点的大型渲染场，Ubuntu Server 都能提供性价比最佳的横向扩展能力。</p>
-        <p><a class="p-button--positive" href="/download">下载 Ubuntu Server</a></p>
+<style>
+.p-strip--suru {
```

Comment:
Rather than duplicate this code (the suru strips are also in `templates/internet-of-things/index.html`) this should be in a sass file (I suggest copying the one from the ubuntu.com project: https://github.com/canonical-web-and-design/ubuntu.com/blob/master/static/sass/_pattern_strip.scss).